services/frontend-v3/src/components/profileForms/welcome/WelcomeView.jsx

```diff
@@ -94,7 +96,7 @@ const WelcomeView = ({ gedsProfiles, intl, load, userId, history }) => {
       if (value) {
         // create profile
         await axios
-          .post(`api/profile/${userId}`, value)
+          .put(`${backendAddress}api/profile/${userId}?language=ENGLISH`, value)
```

Comment:
Here as well. Please use the preconfigured axios instance.

Reply:
Now use axios-instance